### ISSUE TITLE
False positive for no-jquery-raw-elements by complex CSS

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [microsoft/tslint-microsoft-contrib#154]
The following is triggering a failure:

``` typescript
$(StringUtilities.format("div#area_{0}, #area_{0} > span", id))
```

If you don't mind @HamletDRC I can take a look at it soon :smile:. It's a small enough edge case that I wouldn't consider it high priority at all.

**Comment by HamletDRC** [comment]
What do you think the logic should be to detect this?

**Comment by JoshuaKGoldberg** [comment]
Hard to say. There's no exact reliable way to tell the difference between these two:

```typescript
let parent = "div";
parent += "#parent";
let child = "span";

$(parent + " > " + child);
```

```typescript
let left = "<div";
left += " id='parent'";
let right = "</div>";

$(left + " > " + right);
```

...but the latter feels less likely to be common.

How about if the string has no variables added before a `">"` it's considered CSS? Seems a bit hacky but gets rid of the common case.

**Comment by JoshuaKGoldberg** [comment]
### 💀 _**It's time!**_ 💀

**TSLint is being deprecated**; therefore, it and tslint-microsoft-contrib are no longer accepting pull requests for major new changes or features. See https://github.com/palantir/tslint/issues/4534. 😱

If you'd like to see this change implemented, you have two choices:
* **Recommended:** Check if this is available in [ESLint](https://eslint.org) + [typescript-eslint](https://typescript-eslint.io) ✅
* _Not Recommended: Fork TSLint or tslint-microsoft-contrib locally_ 🤷‍♂️

👋 It was a pleasure open sourcing with you!

_If you believe this message was posted here in error, please comment so we can re-open the issue!_
